deployment/E_heroku.md

```diff
@@ -1 +1,277 @@
+- Make our project Heroku-ready
+- Deploy!
+
+## Sign up for Heroku and install the Toolbelt
+
+Signing up to Heroku is very simple, just head over to [https://signup.heroku.com/www-header](https://signup.heroku.com/www-header).
+
+Once this is done download the Heroku Toolbert installer for your platform at [https://toolbelt.heroku.com/](https://toolbelt.heroku.com/).
+
+This will allow you to access Heroku commands from your shell.
+
+## Creating the Heroku application
+
+Now that we have the Toolbelt installed, let's create the Heroku application. In your project directory, run:
+
+```console
+$ heroku create --buildpack "https://github.com/HashNuke/heroku-buildpack-elixir.git"
+Creating mysterious-meadow-6277... done, stack is cedar-14
+Buildpack set. Next release on mysterious-meadow-6277 will use https://github.com/HashNuke/heroku-buildpack-elixir.git.
+https://mysterious-meadow-6277.herokuapp.com/ | https://git.heroku.com/mysterious-meadow-6277.git
+Git remote heroku added
+```
+
+The option we are passing allows us to specify the [Elixir buildpack](https://github.com/HashNuke/heroku-buildpack-elixir) we want Heroku to use.
+A [buildpack](https://devcenter.heroku.com/articles/buildpacks) is a collection of scripts that Heroku uses to prepare your code for execution.
+
+## Adding the Phoenix static buildpack
+
+In order to successfully deploy a Phoenix app, static assets need to be compiled. Let's add the [Phoenix static buildpack](https://github.com/gjaldon/heroku-buildpack-phoenix-static) that is going to take care of that for us.
+
+```console
+$ heroku buildpacks:add https://github.com/gjaldon/phoenix-static-buildpack
+Buildpack added. Next release on mysterious-meadow-6277 will use:
+  1. https://github.com/HashNuke/heroku-buildpack-elixir.git
+  2. https://github.com/gjaldon/phoenix-static-buildpack
+Run `git push heroku master` to create a new release using these buildpacks.
+```
+
+## Making our project Heroku-ready
+
+By default, every new Phoenix project ships with a config file `config/prod.secret.exs` that is used to store configuration that should not be commited along with your source code to your repository.
+
+This works great except Heroku [uses environment variables](https://devcenter.heroku.com/articles/config-vars) to pass sensitive informations to our application. It means we need to make some changes to our config before we can deploy.
+
+First, let's copy over the content of `config/prod.secret.exs` to `config/prod.exs`.
+
+Now we need to replace the sensitive values by a tuple `{:system, "MY_CONFIG"}` which will tell Phoenix to load this configuration from an environment variable.
+
+In our case:
+
+```elixir
+config :phoenix_heroku, PhoenixHeroku.Endpoint,
+  secret_key_base: "my_secret_key"
+```
+
+becomes:
+
+```elixir
+config :phoenix_heroku, PhoenixHeroku.Endpoint,
+  secret_key_base: {:system, "SECRET_KEY_BASE"}
+```
+
+and:
+
+```elixir
+config :phoenix_heroku, PhoenixHeroku.Repo,
```

Comment:
Please switch: `:phoenix_heroku` => `:hello_phoenix` and `PhoenixHeroku` => `HelloPhoenix`